README.md

```diff
@@ -47,20 +47,24 @@ cordova platform add --link /path/to/cordova-ios
 
 ## Updating a Cordova project
 
-When you install a new `cordova-cli` version that comes with a new iOS platform version, from within your project:
+When you install a new version of the [`Cordova CLI`](https://www.npmjs.com/package/cordova) that pins a new version of the [`Cordova-iOS`](https://www.npmjs.com/package/cordova-ios) platform, you can follow these simple upgrade steps within your project:
 
 ```bash
 cordova platform rm ios
 cordova platform add ios
 ```
 
+## Debugging in Xcode
+
+Import project in Xcode through _File > Open_ and targeting `/path/to/your-cdv-project/platforms/ios/`.
```

Comment:
You can also open the `PROJECTNAME.xcodeproj` or ``PROJECTNAME.xcodeworkspace` files in the platforms folder.

Reply:
```suggestion
Import project in Xcode through _File > Open_ and targeting `/path/to/your-cdv-project/platforms/ios/PROJECT_NAME.xcodeworkspace`.
```